src/components/shared/specPropEditor/SpecPropInvalidSetting.tsx

```diff
@@ -37,7 +40,14 @@ function SpecPropInvalidSetting({
                 size="small"
                 sx={{ maxWidth: 'fit-content' }}
                 variant="text"
-                onClick={() => updateDraftedSetting()}
+                onClick={() => {
+                    logRocketEvent('ResetInvalidSetting', {
+                        messageId: invalidSettingsMessageId,
+                        currentSetting: safeCurrentSetting,
+                    });
```

Comment:
I want to track when people click this button to see just how often anyone even sees it.